app/views/games/play.html.erb

```diff
@@ -15,6 +15,7 @@
       </div>
       <div class='right'>
         <div class="timer">
+          <div hidden id="end-time"><%= @game.end %></div>
```

Comment:
we might end up using this; I implemented mine a little differently by taking into account when team was created, but this seems simpler. Can test both